realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -895,6 +816,7 @@ public void onChange(RealmResults<AllTypes> object) {
 
     @Test
     @RunTestInLooperThread
+    @Ignore
```

Comment:
added TODO. We need to recheck this test after integrate OS object notification.